benchmarks/esmf/gridcomp_creation/GridCompCreator.F90

```diff
@@ -0,0 +1,124 @@
+#define RC_ rc=status
+module grid_comp_creator
+
+   use ESMF
+   use shared_constants
+   use, intrinsic :: iso_fortran_env, only: I64 => int64, R64 => real64
+
+   implicit none
+   private
+
+   public :: initialize
+   public :: run
+
+   type :: GC_Container
+      type(ESMF_GridComp) :: gc
+   end type GC_Container
```

Comment:
This intermediate type seems unnecessary.  You can have an array of ESMF_GridComps.

Reply:
Eliminated.